### DISCUSSION TITLE
how to use????????????????????

### DISCUSSION THREAD

**Discussion by BenPlays470** [MoreMcmeta/core#111] [Help Wanted]
I don't know how this mod works, I don't know how to program or anything so I don't know how to use it, and there is no tutorial for this mod, nor does the page explain it, I wanted to use it to leave a specific trim of the trimmable tools shiny but I don't know how to do it :(, if anyone is willing to do or teach, I wanted to leave the sentry trim for sword, shiny

**Comment by soir20** [comment]
There's a demo resource pack available here to use as an example: https://www.curseforge.com/minecraft/texture-packs/moremcmeta-demo

There's also full documentation about how to use the [animation](https://github.com/MoreMcmeta/core/wiki/User-Docs:-Animation-Format) and [emissive](https://github.com/MoreMcmeta/core/wiki/User-Docs:-Emissive-Format) features.

MoreMcmeta works the same way as vanilla `.mcmeta` animations, except you use the `.moremcmeta` file extension instead of `.mcmeta`. So any resource pack tutorial for vanilla block animations would help. MoreMcmeta also supports OptiFine-style animations, so you can also look up a tutorial about how to create an animated resource pack compatible with OptiFine, and those resource packs will work with MoreMcmeta.

**Reply by BenPlays470** [reply]
complex, I confess. but thank you
